sdk/src/index.d.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+/* tslint:disable */
+import * as BigNum from '@liskhq/bignum';
+import * as cryptography from '@liskhq/lisk-cryptography';
+import * as transactions from '@liskhq/lisk-transactions';
+
+declare const Application: any;
```

Comment:
Maybe not now but I would add also the `run` method and `registerTransactions` methods here as are the ones that are used the most.

Reply:
I think defining the type would be bit hard without complete typescript